_posts/en/posts/2016-02-26-zkcp.md

```diff
@@ -0,0 +1,140 @@
+---
+type: posts
+layout: post
+lang: en
+id: en-zkcp-announce
+name: zkcp-announce
+title: The first successful Zero-Knowledge Contingent Payment
+permalink: /en/2016/02/26/zkcp-announce/
+author: gmaxwell
+tags: [privacy]
+version: 1
+excerpt: Announcement for the first Zero-Knowledge Contingent Payment on the Bitcoin Network.
+---
+I am happy to announce the first successful  Zero-Knowledge Contingent Payment (ZKCP) on the Bitcoin Network.
+
+ZKCP is a transaction protocol that allows a buyer to purchase information from a seller using Bitcoin in a manner which is private, scalable, secure, and which doesn't require trusting anyone: the expected information is transferred if and _only_ if the payment is made. The buyer and seller do not need to trust each other or depend on arbitration by a third party.
+
+Imagine a movie-style "briefcase swap" (one party with a briefcase full of cash, another containing secret documents), but without the potential scenario of one of the cases being filled with shredded newspaper and the resulting exciting chase scene.
+
+An example application would be the owners of a particular make of e-book reader cooperating to purchase the DRM master keys from a failing manufacturer, so that they could load their own documents on their readers after the vendor's servers go offline. This type of sale is inherently
+irreversible, potentially crosses multiple jurisdictions, and involves parties whose financial stability is uncertain--meaning that both parties either take a great deal of risk or have to make difficult arrangement. Using a ZKCP avoids the significant transactional costs involved in a
+sale which can otherwise easily go wrong. 
+
+In today's transaction I purchased a solution to a 16x16 Sudoku puzzle for 0.10 BTC  from Sean Bowe, a member of the Zcash team, as part of a demonstration performed live at Financial Cryptography 2016 in Barbados. I played my part in the transaction remotely from California.
+
+The transfer involved two transactions: 
+
+- [FIXME](https://www.blocktrail.com/BTC/tx/FIXME)
+- [FIXME](https://www.blocktrail.com/BTC/tx/FIXME)
```

Comment:
8e5df5f792ac4e98cca87f10aba7947337684a5a0a7333ab897fb9c9d616ba9e and 200554139d1e3fe6e499f6ffb0b6e01e706eb8c897293a7f6a26d25e39623fae